common/CMakeLists.txt

```diff
@@ -171,9 +171,45 @@ set(kissfft_srcs
   src/fft/kiss_fftr.c
 )
 
+set(executor_incs
+  include/pcl/experimental/executor/executor.h
+  include/pcl/experimental/executor/property.h
+  include/pcl/experimental/executor/type_trait.h
+)
+
+set(executor_default_incs_impl
+  include/pcl/experimental/executor/default/cuda_executor.hpp
+  include/pcl/experimental/executor/default/omp_executor.hpp
+  include/pcl/experimental/executor/default/inline_executor.hpp
+  include/pcl/experimental/executor/default/sse_executor.hpp
+)
+
+set(executor_property_incs_impl
+  include/pcl/experimental/executor/property/base_property.hpp
+  include/pcl/experimental/executor/property/allocator.hpp
+  include/pcl/experimental/executor/property/blocking.hpp
+)
+
+set(executor_trait_incs_impl
+  include/pcl/experimental/executor/trait/common_traits.hpp
+  include/pcl/experimental/executor/trait/can_prefer.hpp
+  include/pcl/experimental/executor/trait/can_query.hpp
+  include/pcl/experimental/executor/trait/can_require.hpp
+  include/pcl/experimental/executor/trait/is_executor.hpp
+  include/pcl/experimental/executor/trait/is_executor_available.hpp
+  include/pcl/experimental/executor/trait/is_instance_of_base.hpp
+  include/pcl/experimental/executor/trait/index.hpp
+  include/pcl/experimental/executor/trait/shape.hpp
+)
+
+
+set(executor_srcs
+  include/pcl/experimental/executor/blocking.cpp
+)
+
 set(LIB_NAME "pcl_${SUBSYS_NAME}")
 include_directories("${CMAKE_CURRENT_SOURCE_DIR}/include")
-PCL_ADD_LIBRARY(${LIB_NAME} COMPONENT ${SUBSYS_NAME} SOURCES ${srcs} ${kissfft_srcs} ${incs} ${common_incs} ${impl_incs} ${tools_incs} ${kissfft_incs} ${common_incs_impl} ${range_image_incs} ${range_image_incs_impl})
+PCL_ADD_LIBRARY(${LIB_NAME} COMPONENT ${SUBSYS_NAME} SOURCES ${srcs} ${kissfft_srcs} ${executor_srcs} ${incs} ${common_incs} ${impl_incs} ${tools_incs} ${kissfft_incs} ${common_incs_impl} ${range_image_incs} ${range_image_incs_impl} ${executor_incs})
```

Comment:
Don't add includes to the sources, despite what you're seeing here.

Reply:
Removed